// @marcelinodzn/ds-react/dist/components/Button/index.js
"use strict";
var __create = Object.create;
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __getProtoOf = Object.getPrototypeOf;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toESM = (mod, isNodeMode, target) => (target = mod != null ? __create(__getProtoOf(mod)) : {}, __copyProps(
  // If the importer is in node compatibility mode or this is not an ESM
  // file that has been converted to a CommonJS file using a Babel-
  // compatible transform (i.e. "__esModule" has not been set), then set
  // "default" to the CommonJS "module.exports" for node compatibility.
  isNodeMode || !mod || !mod.__esModule ? __defProp(target, "default", { value: mod, enumerable: true }) : target,
  mod
));
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/components/Button/index.ts
var Button_exports = {};
__export(Button_exports, {
  Button: () => Button,
  default: () => Button_default,
  determineButtonState: () => determineButtonState,
  resolveButtonStyles: () => resolveButtonStyles
});
module.exports = __toCommonJS(Button_exports);

// src/components/Button/Button.tsx
var import_react5 = require("react");
var import_button = require("@react-aria/button");
var import_focus = require("@react-aria/focus");
var import_utils = require("@react-aria/utils");

// src/contexts/SurfaceContext.tsx
var import_react = __toESM(require("react"));
var import_jsx_runtime = (
  // Fragments cannot accept className or style
  // Fragments cannot accept className or style
  require("react/jsx-runtime")
);
var SurfaceContext = (0, import_react.createContext)({
  level: 0,
  hasBoldBackground: false,
  hasColoredBackground: false,
  hasLowAttention: false,
  increment: () => 1
});
function useSurfaceContext() {
  return (0, import_react.useContext)(SurfaceContext);
}
function useSurfaceLevel() {
  const { level } = useSurfaceContext();
  return level;
}
function useHasBoldBackground() {
  const { hasBoldBackground } = useSurfaceContext();
  return hasBoldBackground;
}
function SurfaceProvider({
  level = 0,
  hasBoldBackground = false,
  hasColoredBackground = false,
  hasLowAttention = false,
  appearance,
  children,
  className,
  style,
  as: Component = "div"
}) {
  const contextValue = {
    level,
    hasBoldBackground,
    hasColoredBackground,
    hasLowAttention,
    appearance,
    increment: () => {
      const nextLevel = level + 1;
      return nextLevel > 2 ? 2 : nextLevel;
    }
  };
  const isFragment = Component === import_react.default.Fragment;
  return /* @__PURE__ */ (0, import_jsx_runtime.jsx)(SurfaceContext.Provider, { value: contextValue, children: isFragment ? (
    // Fragments cannot accept className or style
    // Fragments cannot accept className or style
    /* @__PURE__ */ (0, import_jsx_runtime.jsx)(import_react.default.Fragment, { children })
  ) : /* @__PURE__ */ (0, import_jsx_runtime.jsx)(Component, { className, style, children }) });
}

// src/contexts/DsProvider.tsx
var import_react2 = require("react");
var import_ds_tokens = require("@marcelinodzn/ds-tokens");
var import_jsx_runtime2 = require("react/jsx-runtime");
var DsContext = (0, import_react2.createContext)(void 0);
function useDsContextOptional() {
  return (0, import_react2.useContext)(DsContext);
}

// src/hooks/useMotion.ts
var import_react3 = require("react");
function useMotion() {
  const [prefersReducedMotion, setPrefersReducedMotion] = (0, import_react3.useState)(false);
  (0, import_react3.useEffect)(() => {
    if (typeof window === "undefined" || !window.matchMedia) {
      return;
    }
    const mediaQuery = window.matchMedia("(prefers-reduced-motion: reduce)");
    setPrefersReducedMotion(mediaQuery.matches);
    const handler = (event) => {
      setPrefersReducedMotion(event.matches);
    };
    if (mediaQuery.addEventListener) {
      mediaQuery.addEventListener("change", handler);
    } else {
      mediaQuery.addListener(handler);
    }
    return () => {
      if (mediaQuery.removeEventListener) {
        mediaQuery.removeEventListener("change", handler);
      } else {
        mediaQuery.removeListener(handler);
      }
    };
  }, []);
  const motionLevel = prefersReducedMotion ? "subtle" : "moderate";
  return {
    motionLevel,
    prefersReducedMotion
  };
}

// src/components/Label/Label.tsx
var import_react4 = require("react");

// src/components/Label/Label.styles.ts
var import_ds_tokens2 = require("@marcelinodzn/ds-tokens");
function resolveLabelStyles(context) {
  const {
    size,
    weight,
    color,
    attention,
    tinted,
    onMedia,
    align,
    platform = import_ds_tokens2.PLATFORM_MODES.DESKTOP_1440,
    theme,
    density = import_ds_tokens2.DENSITY_MODES.DEFAULT,
    colorMode = import_ds_tokens2.COLOR_MODE_MODES.LIGHT
  } = context;
  const tokenContextBase = {
    [import_ds_tokens2.COLLECTION_NAMES.PLATFORM]: platform,
    [import_ds_tokens2.COLLECTION_NAMES.DENSITY]: density,
    [import_ds_tokens2.COLLECTION_NAMES.COLOR_MODE]: colorMode
  };
  if (theme) {
    const pack1Themes = ["MyJio", "JioHome", "JioMart", "JioFinance"];
    const themePackCollection = pack1Themes.includes(theme) ? import_ds_tokens2.COLLECTION_NAMES.THEME_PACK1 : import_ds_tokens2.COLLECTION_NAMES.THEME_PACK2;
    tokenContextBase[import_ds_tokens2.COLLECTION_NAMES.THEME] = "\u2193Pack1";
    tokenContextBase[themePackCollection] = theme;
  }
  const tokenContext = (0, import_ds_tokens2.createTokenContext)(tokenContextBase);
  const typography = resolveLabelTypography({ size, weight }, tokenContext);
  const textColor = resolveLabelColor(
    {
      color,
      attention,
      tinted,
      onMedia,
      hasBoldBackground: context.hasBoldBackground
    },
    tokenContext
  );
  let opacity = 1;
  if (!tinted) {
    if (color) {
      if (color.includes("medium")) {
        opacity = 0.8;
      } else if (color.includes("low")) {
        opacity = 0.65;
      }
    } else if (attention) {
      const opacityMap = {
        high: 1,
        medium: 0.8,
        low: 0.65,
        "on-bold-high": 1,
        "on-bold-medium": 0.8
      };
      opacity = opacityMap[attention] ?? 1;
    }
  }
  const rootStyles = {
    // Typography
    fontSize: typography.fontSize,
    fontWeight: typography.fontWeight,
    lineHeight: typography.lineHeight,
    fontFamily: typography.fontFamily,
    letterSpacing: typography.letterSpacing,
    // Color
    color: typeof textColor === "string" ? textColor : void 0,
    opacity,
    // Alignment
    textAlign: align,
    // Disable ligatures (multiple properties for browser compatibility)
    fontVariantLigatures: "none",
    fontFeatureSettings: '"liga" 0, "clig" 0',
    WebkitFontFeatureSettings: '"liga" 0, "clig" 0',
    // Reset
    margin: 0,
    padding: 0
  };
  return {
    root: rootStyles
  };
}
function resolveLabelTypography(config, tokenContext) {
  const { size, weight } = config;
  const fontSizeTokenName = `Typography/Size/Label/${size}`;
  const fontSizeValue = (0, import_ds_tokens2.getVariableByName)(fontSizeTokenName, tokenContext);
  const fontSize = fontSizeValue !== null ? `${fontSizeValue}px` : void 0;
  const weightTokenNameMap = {
    low: "Typography/Font Weight/Label Low",
    medium: "Typography/Font Weight/Label Medium",
    high: "Typography/Font Weight/Label High"
  };
  const weightTokenName = weightTokenNameMap[weight];
  const weightValue = (0, import_ds_tokens2.getVariableByName)(weightTokenName, tokenContext);
  const fontWeight = weightValue !== null ? Number(weightValue) : void 0;
  const lineHeight = 1;
  const fontFamilyValue = (0, import_ds_tokens2.getVariableByName)("Typography/Font Family", tokenContext);
  const fontFamily = fontFamilyValue !== null ? String(fontFamilyValue) : "JioType Var, sans-serif";
  const letterSpacing = void 0;
  return {
    fontSize,
    fontWeight,
    lineHeight,
    fontFamily,
    letterSpacing
  };
}
function resolveLabelColor(config, tokenContext) {
  const { color, attention = "high", tinted, onMedia, hasBoldBackground } = config;
  let tokenName;
  let resolutionContext = tokenContext;
  if (color) {
    let effectiveColor = color;
    if (hasBoldBackground && !color.startsWith("on-bold")) {
      effectiveColor = "on-bold-high";
    }
    const colorTokenMap = {
      high: "Text/High",
      medium: "Text/Medium",
      low: "Text/Low",
      "high-tinted": "Text/High Tinted",
      "medium-tinted": "Text/Medium Tinted",
      "low-tinted": "Text/Low Tinted",
      "on-bold-high": "Text/[On Bold] High"
    };
    tokenName = colorTokenMap[effectiveColor];
  } else {
    let level = attention;
    if (attention === "on-bold-high") level = "high";
    if (attention === "on-bold-medium") level = "medium";
    const capLevel = level.charAt(0).toUpperCase() + level.slice(1);
    if (tinted) {
      if (hasBoldBackground || attention.startsWith("on-bold")) {
        tokenName = `Primary/Text/[On Bold] ${capLevel} Tinted`;
      } else {
        tokenName = `Primary/Text/${capLevel} Tinted`;
      }
    } else if (onMedia) {
      tokenName = "Grey/2500";
    } else {
      if (hasBoldBackground || attention.startsWith("on-bold")) {
        if (level === "low") {
          tokenName = `Text/[On Bold] Medium`;
        } else {
          tokenName = `Text/[On Bold] ${capLevel}`;
        }
      } else {
        tokenName = `Text/${capLevel}`;
      }
    }
  }
  const colorValue = (0, import_ds_tokens2.getVariableByName)(tokenName, resolutionContext);
  if (colorValue === null) {
    console.error(
      "[Label.styles] TOKEN_UNRESOLVED: Label color token could not be resolved",
      { color, attention, tinted, onMedia, tokenName, tokenContext: resolutionContext }
    );
    return "currentColor";
  }
  return colorValue;
}

// src/components/Label/Label.tsx
var Label = (0, import_react4.forwardRef)(function Label2(props, forwardedRef) {
  const {
    // Design System props
    size = "M",
    weight = "medium",
    color,
    attention = "high",
    tinted,
    onMedia,
    align = "left",
    as = "label",
    // Content
    children,
    // HTML/React props
    className,
    style: customStyle,
    id,
    htmlFor,
    "aria-label": ariaLabel,
    "aria-describedby": ariaDescribedby,
    "aria-live": ariaLive,
    role,
    title,
    ...restProps
  } = props;
  const dsContext = useDsContextOptional();
  const hasBoldBackground = useHasBoldBackground();
  const resolvedStyles = resolveLabelStyles({
    size,
    weight,
    color,
    attention,
    tinted,
    onMedia,
    hasBoldBackground,
    align,
    platform: dsContext?.platform,
    theme: dsContext?.theme,
    colorMode: dsContext?.colorMode,
    density: dsContext?.density
  });
  const finalRootStyle = customStyle ? { ...resolvedStyles.root, ...customStyle } : resolvedStyles.root;
  const elementProps = {
    ref: forwardedRef,
    className,
    style: finalRootStyle,
    id,
    "aria-label": ariaLabel,
    "aria-describedby": ariaDescribedby,
    "aria-live": ariaLive,
    role,
    title,
    ...restProps
  };
  if (as === "label" && htmlFor) {
    elementProps.htmlFor = htmlFor;
  }
  return (0, import_react4.createElement)(as, elementProps, children);
});
Label.displayName = "Label";

// src/components/Button/Button.styles.ts
var import_ds_tokens3 = require("@marcelinodzn/ds-tokens");
function resolveButtonStyles(context) {
  const {
    size = "M",
    attention,
    appearance,
    contained,
    condensed,
    single,
    fullWidth,
    state,
    platform = import_ds_tokens3.PLATFORM_MODES.DESKTOP_1440,
    theme,
    density = import_ds_tokens3.DENSITY_MODES.DEFAULT,
    colorMode = import_ds_tokens3.COLOR_MODE_MODES.LIGHT,
    disabled = false,
    loading = false,
    isFocusVisible = false,
    motionLevel = "moderate",
    isPressed = false
  } = context;
  const tokenContextBase = {
    [import_ds_tokens3.COLLECTION_NAMES.PLATFORM]: platform,
    [import_ds_tokens3.COLLECTION_NAMES.DENSITY]: density,
    [import_ds_tokens3.COLLECTION_NAMES.COLOR_MODE]: colorMode
  };
  if (theme) {
    const pack1Themes = ["MyJio", "JioHome", "JioMart", "JioFinance"];
    const themePackCollection = pack1Themes.includes(theme) ? import_ds_tokens3.COLLECTION_NAMES.THEME_PACK1 : import_ds_tokens3.COLLECTION_NAMES.THEME_PACK2;
    tokenContextBase[import_ds_tokens3.COLLECTION_NAMES.THEME] = "\u2193Pack1";
    tokenContextBase[themePackCollection] = theme;
  }
  const tokenContext = (0, import_ds_tokens3.createTokenContext)(tokenContextBase);
  const borderRadius = (0, import_ds_tokens3.getVariableByName)("Shape/Pill", tokenContext);
  const padding = resolvePadding(
    { size, contained, condensed, single },
    tokenContext
  );
  const colors = resolveColors(
    { attention, appearance, contained, state, colorMode },
    tokenContext
  );
  const itemSpacing = (0, import_ds_tokens3.getVariableByName)("Spacing/2XS", tokenContext);
  const opacity = disabled || loading ? 0.3 : 1;
  const calculateScale = () => {
    if (!isPressed || motionLevel === "subtle") {
      return 1;
    }
    if (single) {
      return 0.93;
    }
    if (fullWidth) {
      return 0.99;
    }
    return 0.97;
  };
  const scale = calculateScale();
  const transitionProperties = ["background-color", "color", "opacity"];
  if (motionLevel === "moderate") {
    transitionProperties.push("transform");
  }
  const transition = (0, import_ds_tokens3.createTransition)(transitionProperties, "m", "transition", motionLevel);
  const rootStyles = {
    // Layout
    display: "inline-flex",
    flexDirection: "row",
    alignItems: "center",
    justifyContent: "center",
    gap: itemSpacing !== null ? `${itemSpacing}px` : void 0,
    // Sizing
    width: fullWidth ? "100%" : "auto",
    // Min height is fixed at dimension.L for contained buttons (not dependent on size)
    minHeight: !contained ? void 0 : (() => {
      const minHeightToken = (0, import_ds_tokens3.getVariableByName)("Dimension/L", tokenContext);
      return minHeightToken !== null ? `${minHeightToken}px` : void 0;
    })(),
    // Spacing
    ...padding,
    // Shape
    borderRadius: borderRadius !== null ? `${borderRadius}px` : void 0,
    // Colors
    backgroundColor: typeof colors.background === "string" ? colors.background : void 0,
    color: typeof colors.text === "string" ? colors.text : void 0,
    // State
    opacity,
    cursor: disabled || loading ? "not-allowed" : "pointer",
    pointerEvents: disabled || loading ? "none" : "auto",
    // Motion: Scale transform on press (only for moderate motion level)
    transform: `scale(${scale})`,
    // Typography is now handled by Label component (delegated)
    // No fontSize, fontWeight, lineHeight, or fontFamily needed here
    // Reset
    border: "none",
    outline: isFocusVisible ? `2px solid ${colors.focus}` : "none",
    outlineOffset: "2px",
    textDecoration: resolveTextDecoration(attention, contained, state),
    // Transitions - using motion tokens (M duration, transition easing)
    transition,
    // User select
    userSelect: "none",
    WebkitUserSelect: "none",
    WebkitTapHighlightColor: "transparent"
  };
  const contentMinHeightMap = {
    "XS": "Dimension/S",
    "S": "Dimension/M",
    "M": "Dimension/L",
    "L": "Dimension/XL",
    "XL": "Dimension/2XL"
  };
  const contentMinHeightToken = (0, import_ds_tokens3.getVariableByName)(contentMinHeightMap[size], tokenContext);
  const contentMinHeight = contentMinHeightToken !== null ? `${contentMinHeightToken}px` : void 0;
  const contentStyles = {
    display: "inline-flex",
    alignItems: "center",
    justifyContent: "center",
    lineHeight: 1,
    // Numeric value for exact 100% line-height
    verticalAlign: "middle",
    // Ensures content aligns properly within inline-flex
    minHeight: contentMinHeight
    // Matches icon dimension for this button size
  };
  const slotStyles = {
    display: "inline-flex",
    alignItems: "center",
    justifyContent: "center",
    lineHeight: "0"
  };
  return {
    root: rootStyles,
    start: slotStyles,
    content: contentStyles,
    end: slotStyles
  };
}
function resolvePadding(config, tokenContext) {
  const { size, contained, condensed, single } = config;
  if (!contained) {
    return {};
  }
  const horizontalPaddingMap = {
    "XS": "Spacing/S",
    "S": "Spacing/M",
    "M": "Spacing/L",
    "L": "Spacing/XL",
    "XL": "Spacing/2XL"
  };
  const verticalPaddingMap = {
    "XS": "Spacing/2XS",
    "S": "Spacing/XS",
    "M": "Spacing/S",
    "L": "Spacing/M",
    "XL": "Spacing/L"
  };
  if (single) {
    if (condensed) {
      const padding = (0, import_ds_tokens3.getVariableByName)("Spacing/2XS", tokenContext);
      return {
        paddingLeft: padding !== null ? `${padding}px` : void 0,
        paddingRight: padding !== null ? `${padding}px` : void 0,
        paddingTop: padding !== null ? `${padding}px` : void 0,
        paddingBottom: padding !== null ? `${padding}px` : void 0
      };
    } else {
      const verticalToken = verticalPaddingMap[size] || "Spacing/S";
      const padding = (0, import_ds_tokens3.getVariableByName)(verticalToken, tokenContext);
      return {
        paddingLeft: padding !== null ? `${padding}px` : void 0,
        paddingRight: padding !== null ? `${padding}px` : void 0,
        paddingTop: padding !== null ? `${padding}px` : void 0,
        paddingBottom: padding !== null ? `${padding}px` : void 0
      };
    }
  }
  if (condensed) {
    const horizontalSpacing = (0, import_ds_tokens3.getVariableByName)("Spacing/XS", tokenContext);
    const verticalSpacing = (0, import_ds_tokens3.getVariableByName)("Spacing/2XS", tokenContext);
    return {
      paddingLeft: horizontalSpacing !== null ? `${horizontalSpacing}px` : void 0,
      paddingRight: horizontalSpacing !== null ? `${horizontalSpacing}px` : void 0,
      paddingTop: verticalSpacing !== null ? `${verticalSpacing}px` : void 0,
      paddingBottom: verticalSpacing !== null ? `${verticalSpacing}px` : void 0
    };
  } else {
    const horizontalToken = horizontalPaddingMap[size] || "Spacing/L";
    const verticalToken = verticalPaddingMap[size] || "Spacing/S";
    const horizontalPadding = (0, import_ds_tokens3.getVariableByName)(horizontalToken, tokenContext);
    const verticalPadding = (0, import_ds_tokens3.getVariableByName)(verticalToken, tokenContext);
    return {
      paddingLeft: horizontalPadding !== null ? `${horizontalPadding}px` : void 0,
      paddingRight: horizontalPadding !== null ? `${horizontalPadding}px` : void 0,
      paddingTop: verticalPadding !== null ? `${verticalPadding}px` : void 0,
      paddingBottom: verticalPadding !== null ? `${verticalPadding}px` : void 0
    };
  }
}
function resolveColors(config, tokenContext) {
  const { attention, appearance, contained, colorMode } = config;
  const figmaAppearance = appearance === "auto" ? "Primary" : appearance.charAt(0).toUpperCase() + appearance.slice(1);
  const emphasisMap = {
    high: "Bold",
    medium: "Subtle",
    low: "Minimal"
  };
  const emphasis = emphasisMap[attention];
  let background = null;
  if (attention === "low") {
    background = "transparent";
  } else if (contained) {
    const backgroundTokenName = `${figmaAppearance}/Background/${emphasis}`;
    background = (0, import_ds_tokens3.getVariableByName)(backgroundTokenName, tokenContext);
    if (background === null && attention === "high") {
      const a11yTokenName = `${figmaAppearance}/Background/${emphasis} a11y`;
      background = (0, import_ds_tokens3.getVariableByName)(a11yTokenName, tokenContext);
    }
    if (background === null) {
      background = (0, import_ds_tokens3.getVariableByName)(`Neutral/Background/${emphasis}`, tokenContext);
    }
  } else {
    background = "transparent";
  }
  const textLevelMap = {
    high: "High",
    medium: "Medium",
    low: "Low"
  };
  const textLevel = textLevelMap[attention];
  const hasBoldBackground = contained && attention === "high";
  const hasMediumColoredBackground = contained && attention === "medium" && appearance !== "neutral";
  let text = null;
  if (hasBoldBackground) {
    text = (0, import_ds_tokens3.getVariableByName)(`${figmaAppearance}/Text/[On Bold] ${textLevel} Tinted`, tokenContext);
    if (text === null) {
      text = (0, import_ds_tokens3.getVariableByName)(`${figmaAppearance}/Text/[On Bold] ${textLevel}`, tokenContext);
    }
    if (text === null) {
      text = (0, import_ds_tokens3.getVariableByName)(`Neutral/Text/[On Bold] ${textLevel}`, tokenContext);
    }
  } else if (hasMediumColoredBackground) {
    text = (0, import_ds_tokens3.getVariableByName)(`${figmaAppearance}/Text/High Tinted`, tokenContext);
    if (text === null) {
      text = (0, import_ds_tokens3.getVariableByName)("Neutral/Text/High Tinted", tokenContext);
    }
  } else if (attention === "low") {
    text = (0, import_ds_tokens3.getVariableByName)(`${figmaAppearance}/Text/Medium Tinted`, tokenContext);
    if (text === null) {
      text = (0, import_ds_tokens3.getVariableByName)("Neutral/Text/Medium Tinted", tokenContext);
    }
  } else {
    text = (0, import_ds_tokens3.getVariableByName)(`${figmaAppearance}/Text/${textLevel}`, tokenContext);
    if (text === null && contained) {
      text = (0, import_ds_tokens3.getVariableByName)(`${figmaAppearance}/Text/${textLevel} Tinted`, tokenContext);
    }
    if (text === null) {
      text = (0, import_ds_tokens3.getVariableByName)(`Neutral/Text/${textLevel}`, tokenContext);
    }
  }
  if (text === null) {
    const surfaceText = (0, import_ds_tokens3.getVariableByName)("Neutral/Text/High", tokenContext);
    if (surfaceText !== null) {
      text = surfaceText;
    }
  }
  if (text === null) {
    console.error(
      "[Button.styles] TOKEN_UNRESOLVED: Text color token could not be resolved",
      { appearance, attention, textLevel, tokenContext }
    );
    text = colorMode === import_ds_tokens3.COLOR_MODE_MODES.DARK ? "#FFFFFF" : "#000000";
  }
  let focus = (0, import_ds_tokens3.getVariableByName)("Primary/Background/Bold", tokenContext);
  if (focus === null) {
    focus = (0, import_ds_tokens3.getVariableByName)("Primary/Background/Bold a11ly", tokenContext);
  }
  if (focus === null) {
    console.error(
      "[Button.styles] TOKEN_UNRESOLVED: Focus ring color token could not be resolved",
      { tokenContext }
    );
    focus = "#0066FF";
  }
  return { background, text, focus };
}
function resolveTextDecoration(attention, contained, state) {
  if (!contained) {
    if (attention === "high") {
      return "underline";
    }
    if (attention === "medium" && (state === "hover" || state === "pressed" || state === "focus")) {
      return "underline";
    }
    if (attention === "low" && (state === "hover" || state === "pressed" || state === "focus")) {
      return "underline";
    }
  }
  return "none";
}
function determineButtonState(isPressed, isHovered, isFocused, isDisabled, isLoading) {
  if (isDisabled) return "disabled";
  if (isLoading) return "loading";
  if (isPressed) return "pressed";
  if (isHovered) return "hover";
  if (isFocused) return "focus";
  return "idle";
}

// src/components/Button/Button.tsx
var import_jsx_runtime3 = require("react/jsx-runtime");
var Button = (0, import_react5.forwardRef)(
  function Button2(props, forwardedRef) {
    const {
      // Design System props
      size = "M",
      attention = "medium",
      appearance = "auto",
      contained = true,
      condensed = false,
      single = false,
      fullWidth = false,
      loading = false,
      motionLevel: motionLevelProp,
      // Content slots (Slots 2, 3, 4 from blueprint)
      start,
      // Slot 2
      content,
      // Slot 3
      end,
      // Slot 4
      children,
      // Backward compatibility (maps to content)
      // HTML/React props
      className,
      style: customStyle,
      isDisabled,
      onPress,
      // Accessibility
      "aria-label": ariaLabel,
      ...restProps
    } = props;
    const slotContent = content ?? children;
    if (single && !ariaLabel && !loading) {
      console.error(
        "Button: Single buttons (single=true) require aria-label for accessibility"
      );
    }
    if (content && children) {
      console.warn(
        "Button: Both `content` and `children` props provided. Using `content`. Please use only `content` prop to follow the slot system."
      );
    }
    if (condensed && !contained) {
      console.warn(
        "Button: condensed property has no effect when contained=false"
      );
    }
    const buttonRef = forwardedRef || (0, import_react5.useRef)(null);
    const [isHovered, setIsHovered] = (0, import_react5.useState)(false);
    const surfaceLevel = useSurfaceLevel();
    const dsContext = useDsContextOptional();
    const { motionLevel: detectedMotionLevel } = useMotion();
    const motionLevel = motionLevelProp ?? detectedMotionLevel;
    const disabled = isDisabled || loading;
    const hasBoldBackground = contained && attention === "high";
    const hasColoredBackground = contained && attention === "medium" && appearance !== "neutral";
    const { buttonProps, isPressed } = (0, import_button.useButton)(
      {
        ...restProps,
        isDisabled: disabled,
        onPress,
        "aria-label": ariaLabel
      },
      buttonRef
    );
    const { isFocusVisible, focusProps } = (0, import_focus.useFocusRing)();
    const currentState = determineButtonState(
      isPressed,
      isHovered,
      false,
      // isFocused - using isFocusVisible instead
      disabled,
      loading
    );
    const resolvedStyles = resolveButtonStyles({
      size,
      attention,
      appearance,
      contained,
      condensed,
      single,
      fullWidth,
      state: currentState,
      surfaceLevel,
      disabled,
      loading,
      isFocusVisible,
      // Motion system integration
      motionLevel,
      isPressed,
      // Get platform, theme, colorMode, density from context (if available)
      platform: dsContext?.platform,
      theme: dsContext?.theme,
      colorMode: dsContext?.colorMode,
      density: dsContext?.density
    });
    const finalRootStyle = customStyle ? { ...resolvedStyles.root, ...customStyle } : resolvedStyles.root;
    const handleMouseEnter = () => setIsHovered(true);
    const handleMouseLeave = () => setIsHovered(false);
    const buttonTextColor = resolvedStyles.root.color;
    const renderSlot3Content = () => {
      if (loading) {
        return /* @__PURE__ */ (0, import_jsx_runtime3.jsx)("span", { style: resolvedStyles.content, children: /* @__PURE__ */ (0, import_jsx_runtime3.jsx)(
          Label,
          {
            as: "span",
            size,
            weight: "high",
            role: "status",
            "aria-live": "polite",
            style: { color: buttonTextColor },
            children: "Loading..."
          }
        ) });
      }
      if (single && typeof slotContent === "string") {
        return null;
      }
      return /* @__PURE__ */ (0, import_jsx_runtime3.jsx)("span", { style: resolvedStyles.content, children: /* @__PURE__ */ (0, import_jsx_runtime3.jsx)(Label, { as: "span", size, weight: "high", style: { color: buttonTextColor }, children: slotContent }) });
    };
    const buttonContent = /* @__PURE__ */ (0, import_jsx_runtime3.jsxs)(import_jsx_runtime3.Fragment, { children: [
      start && !loading && /* @__PURE__ */ (0, import_jsx_runtime3.jsx)("span", { style: resolvedStyles.start, children: start }),
      renderSlot3Content(),
      end && !loading && /* @__PURE__ */ (0, import_jsx_runtime3.jsx)("span", { style: resolvedStyles.end, children: end })
    ] });
    return /* @__PURE__ */ (0, import_jsx_runtime3.jsx)(
      "button",
      {
        ...(0, import_utils.mergeProps)(buttonProps, focusProps),
        ref: buttonRef,
        className,
        style: finalRootStyle,
        onMouseEnter: handleMouseEnter,
        onMouseLeave: handleMouseLeave,
        "aria-busy": loading,
        type: "button",
        children: /* @__PURE__ */ (0, import_jsx_runtime3.jsx)(
          SurfaceProvider,
          {
            hasBoldBackground,
            hasColoredBackground,
            hasLowAttention: attention === "low",
            appearance,
            as: "span",
            style: { display: "contents" },
            children: buttonContent
          }
        )
      }
    );
  }
);
Button.displayName = "Button";
var Button_default = Button;
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  Button,
  determineButtonState,
  resolveButtonStyles
});
