// @marcelinodzn/ds-react/dist/components/Button/index.mjs
// src/components/Button/Button.tsx
import { forwardRef as forwardRef2, useRef, useState as useState3 } from "react";
import { useButton } from "@react-aria/button";
import { useFocusRing } from "@react-aria/focus";
import { mergeProps } from "@react-aria/utils";

// src/contexts/SurfaceContext.tsx
import React, { createContext, useContext } from "react";
import { jsx } from "react/jsx-runtime";
var SurfaceContext = createContext({
  level: 0,
  hasBoldBackground: false,
  hasColoredBackground: false,
  hasLowAttention: false,
  increment: () => 1
});
function useSurfaceContext() {
  return useContext(SurfaceContext);
}
function useSurfaceLevel() {
  const { level } = useSurfaceContext();
  return level;
}
function useHasBoldBackground() {
  const { hasBoldBackground } = useSurfaceContext();
  return hasBoldBackground;
}
function SurfaceProvider({
  level = 0,
  hasBoldBackground = false,
  hasColoredBackground = false,
  hasLowAttention = false,
  appearance,
  children,
  className,
  style,
  as: Component = "div"
}) {
  const contextValue = {
    level,
    hasBoldBackground,
    hasColoredBackground,
    hasLowAttention,
    appearance,
    increment: () => {
      const nextLevel = level + 1;
      return nextLevel > 2 ? 2 : nextLevel;
    }
  };
  const isFragment = Component === React.Fragment;
  return /* @__PURE__ */ jsx(SurfaceContext.Provider, { value: contextValue, children: isFragment ? (
    // Fragments cannot accept className or style
    // Fragments cannot accept className or style
    /* @__PURE__ */ jsx(React.Fragment, { children })
  ) : /* @__PURE__ */ jsx(Component, { className, style, children }) });
}

// src/contexts/DsProvider.tsx
import { createContext as createContext2, useContext as useContext2, useState, useMemo, useEffect } from "react";
import { createTokenContext, COLLECTION_NAMES } from "@marcelinodzn/ds-tokens";
import { jsx as jsx2 } from "react/jsx-runtime";
var DsContext = createContext2(void 0);
function useDsContextOptional() {
  return useContext2(DsContext);
}

// src/hooks/useMotion.ts
import { useEffect as useEffect2, useState as useState2 } from "react";
function useMotion() {
  const [prefersReducedMotion, setPrefersReducedMotion] = useState2(false);
  useEffect2(() => {
    if (typeof window === "undefined" || !window.matchMedia) {
      return;
    }
    const mediaQuery = window.matchMedia("(prefers-reduced-motion: reduce)");
    setPrefersReducedMotion(mediaQuery.matches);
    const handler = (event) => {
      setPrefersReducedMotion(event.matches);
    };
    if (mediaQuery.addEventListener) {
      mediaQuery.addEventListener("change", handler);
    } else {
      mediaQuery.addListener(handler);
    }
    return () => {
      if (mediaQuery.removeEventListener) {
        mediaQuery.removeEventListener("change", handler);
      } else {
        mediaQuery.removeListener(handler);
      }
    };
  }, []);
  const motionLevel = prefersReducedMotion ? "subtle" : "moderate";
  return {
    motionLevel,
    prefersReducedMotion
  };
}

// src/components/Label/Label.tsx
import { createElement, forwardRef } from "react";

// src/components/Label/Label.styles.ts
import {
  getVariableByName,
  createTokenContext as createTokenContext2,
  COLLECTION_NAMES as COLLECTION_NAMES2,
  COLOR_MODE_MODES,
  PLATFORM_MODES,
  DENSITY_MODES
} from "@marcelinodzn/ds-tokens";
function resolveLabelStyles(context) {
  const {
    size,
    weight,
    color,
    attention,
    tinted,
    onMedia,
    align,
    platform = PLATFORM_MODES.DESKTOP_1440,
    theme,
    density = DENSITY_MODES.DEFAULT,
    colorMode = COLOR_MODE_MODES.LIGHT
  } = context;
  const tokenContextBase = {
    [COLLECTION_NAMES2.PLATFORM]: platform,
    [COLLECTION_NAMES2.DENSITY]: density,
    [COLLECTION_NAMES2.COLOR_MODE]: colorMode
  };
  if (theme) {
    const pack1Themes = ["MyJio", "JioHome", "JioMart", "JioFinance"];
    const themePackCollection = pack1Themes.includes(theme) ? COLLECTION_NAMES2.THEME_PACK1 : COLLECTION_NAMES2.THEME_PACK2;
    tokenContextBase[COLLECTION_NAMES2.THEME] = "\u2193Pack1";
    tokenContextBase[themePackCollection] = theme;
  }
  const tokenContext = createTokenContext2(tokenContextBase);
  const typography = resolveLabelTypography({ size, weight }, tokenContext);
  const textColor = resolveLabelColor(
    {
      color,
      attention,
      tinted,
      onMedia,
      hasBoldBackground: context.hasBoldBackground
    },
    tokenContext
  );
  let opacity = 1;
  if (!tinted) {
    if (color) {
      if (color.includes("medium")) {
        opacity = 0.8;
      } else if (color.includes("low")) {
        opacity = 0.65;
      }
    } else if (attention) {
      const opacityMap = {
        high: 1,
        medium: 0.8,
        low: 0.65,
        "on-bold-high": 1,
        "on-bold-medium": 0.8
      };
      opacity = opacityMap[attention] ?? 1;
    }
  }
  const rootStyles = {
    // Typography
    fontSize: typography.fontSize,
    fontWeight: typography.fontWeight,
    lineHeight: typography.lineHeight,
    fontFamily: typography.fontFamily,
    letterSpacing: typography.letterSpacing,
    // Color
    color: typeof textColor === "string" ? textColor : void 0,
    opacity,
    // Alignment
    textAlign: align,
    // Disable ligatures (multiple properties for browser compatibility)
    fontVariantLigatures: "none",
    fontFeatureSettings: '"liga" 0, "clig" 0',
    WebkitFontFeatureSettings: '"liga" 0, "clig" 0',
    // Reset
    margin: 0,
    padding: 0
  };
  return {
    root: rootStyles
  };
}
function resolveLabelTypography(config, tokenContext) {
  const { size, weight } = config;
  const fontSizeTokenName = `Typography/Size/Label/${size}`;
  const fontSizeValue = getVariableByName(fontSizeTokenName, tokenContext);
  const fontSize = fontSizeValue !== null ? `${fontSizeValue}px` : void 0;
  const weightTokenNameMap = {
    low: "Typography/Font Weight/Label Low",
    medium: "Typography/Font Weight/Label Medium",
    high: "Typography/Font Weight/Label High"
  };
  const weightTokenName = weightTokenNameMap[weight];
  const weightValue = getVariableByName(weightTokenName, tokenContext);
  const fontWeight = weightValue !== null ? Number(weightValue) : void 0;
  const lineHeight = 1;
  const fontFamilyValue = getVariableByName("Typography/Font Family", tokenContext);
  const fontFamily = fontFamilyValue !== null ? String(fontFamilyValue) : "JioType Var, sans-serif";
  const letterSpacing = void 0;
  return {
    fontSize,
    fontWeight,
    lineHeight,
    fontFamily,
    letterSpacing
  };
}
function resolveLabelColor(config, tokenContext) {
  const { color, attention = "high", tinted, onMedia, hasBoldBackground } = config;
  let tokenName;
  let resolutionContext = tokenContext;
  if (color) {
    let effectiveColor = color;
    if (hasBoldBackground && !color.startsWith("on-bold")) {
      effectiveColor = "on-bold-high";
    }
    const colorTokenMap = {
      high: "Text/High",
      medium: "Text/Medium",
      low: "Text/Low",
      "high-tinted": "Text/High Tinted",
      "medium-tinted": "Text/Medium Tinted",
      "low-tinted": "Text/Low Tinted",
      "on-bold-high": "Text/[On Bold] High"
    };
    tokenName = colorTokenMap[effectiveColor];
  } else {
    let level = attention;
    if (attention === "on-bold-high") level = "high";
    if (attention === "on-bold-medium") level = "medium";
    const capLevel = level.charAt(0).toUpperCase() + level.slice(1);
    if (tinted) {
      if (hasBoldBackground || attention.startsWith("on-bold")) {
        tokenName = `Primary/Text/[On Bold] ${capLevel} Tinted`;
      } else {
        tokenName = `Primary/Text/${capLevel} Tinted`;
      }
    } else if (onMedia) {
      tokenName = "Grey/2500";
    } else {
      if (hasBoldBackground || attention.startsWith("on-bold")) {
        if (level === "low") {
          tokenName = `Text/[On Bold] Medium`;
        } else {
          tokenName = `Text/[On Bold] ${capLevel}`;
        }
      } else {
        tokenName = `Text/${capLevel}`;
      }
    }
  }
  const colorValue = getVariableByName(tokenName, resolutionContext);
  if (colorValue === null) {
    console.error(
      "[Label.styles] TOKEN_UNRESOLVED: Label color token could not be resolved",
      { color, attention, tinted, onMedia, tokenName, tokenContext: resolutionContext }
    );
    return "currentColor";
  }
  return colorValue;
}

// src/components/Label/Label.tsx
var Label = forwardRef(function Label2(props, forwardedRef) {
  const {
    // Design System props
    size = "M",
    weight = "medium",
    color,
    attention = "high",
    tinted,
    onMedia,
    align = "left",
    as = "label",
    // Content
    children,
    // HTML/React props
    className,
    style: customStyle,
    id,
    htmlFor,
    "aria-label": ariaLabel,
    "aria-describedby": ariaDescribedby,
    "aria-live": ariaLive,
    role,
    title,
    ...restProps
  } = props;
  const dsContext = useDsContextOptional();
  const hasBoldBackground = useHasBoldBackground();
  const resolvedStyles = resolveLabelStyles({
    size,
    weight,
    color,
    attention,
    tinted,
    onMedia,
    hasBoldBackground,
    align,
    platform: dsContext?.platform,
    theme: dsContext?.theme,
    colorMode: dsContext?.colorMode,
    density: dsContext?.density
  });
  const finalRootStyle = customStyle ? { ...resolvedStyles.root, ...customStyle } : resolvedStyles.root;
  const elementProps = {
    ref: forwardedRef,
    className,
    style: finalRootStyle,
    id,
    "aria-label": ariaLabel,
    "aria-describedby": ariaDescribedby,
    "aria-live": ariaLive,
    role,
    title,
    ...restProps
  };
  if (as === "label" && htmlFor) {
    elementProps.htmlFor = htmlFor;
  }
  return createElement(as, elementProps, children);
});
Label.displayName = "Label";

// src/components/Button/Button.styles.ts
import {
  getVariableByName as getVariableByName2,
  createTokenContext as createTokenContext3,
  COLLECTION_NAMES as COLLECTION_NAMES3,
  COLOR_MODE_MODES as COLOR_MODE_MODES2,
  PLATFORM_MODES as PLATFORM_MODES2,
  DENSITY_MODES as DENSITY_MODES2,
  createTransition
} from "@marcelinodzn/ds-tokens";
function resolveButtonStyles(context) {
  const {
    size = "M",
    attention,
    appearance,
    contained,
    condensed,
    single,
    fullWidth,
    state,
    platform = PLATFORM_MODES2.DESKTOP_1440,
    theme,
    density = DENSITY_MODES2.DEFAULT,
    colorMode = COLOR_MODE_MODES2.LIGHT,
    disabled = false,
    loading = false,
    isFocusVisible = false,
    motionLevel = "moderate",
    isPressed = false
  } = context;
  const tokenContextBase = {
    [COLLECTION_NAMES3.PLATFORM]: platform,
    [COLLECTION_NAMES3.DENSITY]: density,
    [COLLECTION_NAMES3.COLOR_MODE]: colorMode
  };
  if (theme) {
    const pack1Themes = ["MyJio", "JioHome", "JioMart", "JioFinance"];
    const themePackCollection = pack1Themes.includes(theme) ? COLLECTION_NAMES3.THEME_PACK1 : COLLECTION_NAMES3.THEME_PACK2;
    tokenContextBase[COLLECTION_NAMES3.THEME] = "\u2193Pack1";
    tokenContextBase[themePackCollection] = theme;
  }
  const tokenContext = createTokenContext3(tokenContextBase);
  const borderRadius = getVariableByName2("Shape/Pill", tokenContext);
  const padding = resolvePadding(
    { size, contained, condensed, single },
    tokenContext
  );
  const colors = resolveColors(
    { attention, appearance, contained, state, colorMode },
    tokenContext
  );
  const itemSpacing = getVariableByName2("Spacing/2XS", tokenContext);
  const opacity = disabled || loading ? 0.3 : 1;
  const calculateScale = () => {
    if (!isPressed || motionLevel === "subtle") {
      return 1;
    }
    if (single) {
      return 0.93;
    }
    if (fullWidth) {
      return 0.99;
    }
    return 0.97;
  };
  const scale = calculateScale();
  const transitionProperties = ["background-color", "color", "opacity"];
  if (motionLevel === "moderate") {
    transitionProperties.push("transform");
  }
  const transition = createTransition(transitionProperties, "m", "transition", motionLevel);
  const rootStyles = {
    // Layout
    display: "inline-flex",
    flexDirection: "row",
    alignItems: "center",
    justifyContent: "center",
    gap: itemSpacing !== null ? `${itemSpacing}px` : void 0,
    // Sizing
    width: fullWidth ? "100%" : "auto",
    // Min height is fixed at dimension.L for contained buttons (not dependent on size)
    minHeight: !contained ? void 0 : (() => {
      const minHeightToken = getVariableByName2("Dimension/L", tokenContext);
      return minHeightToken !== null ? `${minHeightToken}px` : void 0;
    })(),
    // Spacing
    ...padding,
    // Shape
    borderRadius: borderRadius !== null ? `${borderRadius}px` : void 0,
    // Colors
    backgroundColor: typeof colors.background === "string" ? colors.background : void 0,
    color: typeof colors.text === "string" ? colors.text : void 0,
    // State
    opacity,
    cursor: disabled || loading ? "not-allowed" : "pointer",
    pointerEvents: disabled || loading ? "none" : "auto",
    // Motion: Scale transform on press (only for moderate motion level)
    transform: `scale(${scale})`,
    // Typography is now handled by Label component (delegated)
    // No fontSize, fontWeight, lineHeight, or fontFamily needed here
    // Reset
    border: "none",
    outline: isFocusVisible ? `2px solid ${colors.focus}` : "none",
    outlineOffset: "2px",
    textDecoration: resolveTextDecoration(attention, contained, state),
    // Transitions - using motion tokens (M duration, transition easing)
    transition,
    // User select
    userSelect: "none",
    WebkitUserSelect: "none",
    WebkitTapHighlightColor: "transparent"
  };
  const contentMinHeightMap = {
    "XS": "Dimension/S",
    "S": "Dimension/M",
    "M": "Dimension/L",
    "L": "Dimension/XL",
    "XL": "Dimension/2XL"
  };
  const contentMinHeightToken = getVariableByName2(contentMinHeightMap[size], tokenContext);
  const contentMinHeight = contentMinHeightToken !== null ? `${contentMinHeightToken}px` : void 0;
  const contentStyles = {
    display: "inline-flex",
    alignItems: "center",
    justifyContent: "center",
    lineHeight: 1,
    // Numeric value for exact 100% line-height
    verticalAlign: "middle",
    // Ensures content aligns properly within inline-flex
    minHeight: contentMinHeight
    // Matches icon dimension for this button size
  };
  const slotStyles = {
    display: "inline-flex",
    alignItems: "center",
    justifyContent: "center",
    lineHeight: "0"
  };
  return {
    root: rootStyles,
    start: slotStyles,
    content: contentStyles,
    end: slotStyles
  };
}
function resolvePadding(config, tokenContext) {
  const { size, contained, condensed, single } = config;
  if (!contained) {
    return {};
  }
  const horizontalPaddingMap = {
    "XS": "Spacing/S",
    "S": "Spacing/M",
    "M": "Spacing/L",
    "L": "Spacing/XL",
    "XL": "Spacing/2XL"
  };
  const verticalPaddingMap = {
    "XS": "Spacing/2XS",
    "S": "Spacing/XS",
    "M": "Spacing/S",
    "L": "Spacing/M",
    "XL": "Spacing/L"
  };
  if (single) {
    if (condensed) {
      const padding = getVariableByName2("Spacing/2XS", tokenContext);
      return {
        paddingLeft: padding !== null ? `${padding}px` : void 0,
        paddingRight: padding !== null ? `${padding}px` : void 0,
        paddingTop: padding !== null ? `${padding}px` : void 0,
        paddingBottom: padding !== null ? `${padding}px` : void 0
      };
    } else {
      const verticalToken = verticalPaddingMap[size] || "Spacing/S";
      const padding = getVariableByName2(verticalToken, tokenContext);
      return {
        paddingLeft: padding !== null ? `${padding}px` : void 0,
        paddingRight: padding !== null ? `${padding}px` : void 0,
        paddingTop: padding !== null ? `${padding}px` : void 0,
        paddingBottom: padding !== null ? `${padding}px` : void 0
      };
    }
  }
  if (condensed) {
    const horizontalSpacing = getVariableByName2("Spacing/XS", tokenContext);
    const verticalSpacing = getVariableByName2("Spacing/2XS", tokenContext);
    return {
      paddingLeft: horizontalSpacing !== null ? `${horizontalSpacing}px` : void 0,
      paddingRight: horizontalSpacing !== null ? `${horizontalSpacing}px` : void 0,
      paddingTop: verticalSpacing !== null ? `${verticalSpacing}px` : void 0,
      paddingBottom: verticalSpacing !== null ? `${verticalSpacing}px` : void 0
    };
  } else {
    const horizontalToken = horizontalPaddingMap[size] || "Spacing/L";
    const verticalToken = verticalPaddingMap[size] || "Spacing/S";
    const horizontalPadding = getVariableByName2(horizontalToken, tokenContext);
    const verticalPadding = getVariableByName2(verticalToken, tokenContext);
    return {
      paddingLeft: horizontalPadding !== null ? `${horizontalPadding}px` : void 0,
      paddingRight: horizontalPadding !== null ? `${horizontalPadding}px` : void 0,
      paddingTop: verticalPadding !== null ? `${verticalPadding}px` : void 0,
      paddingBottom: verticalPadding !== null ? `${verticalPadding}px` : void 0
    };
  }
}
function resolveColors(config, tokenContext) {
  const { attention, appearance, contained, colorMode } = config;
  const figmaAppearance = appearance === "auto" ? "Primary" : appearance.charAt(0).toUpperCase() + appearance.slice(1);
  const emphasisMap = {
    high: "Bold",
    medium: "Subtle",
    low: "Minimal"
  };
  const emphasis = emphasisMap[attention];
  let background = null;
  if (attention === "low") {
    background = "transparent";
  } else if (contained) {
    const backgroundTokenName = `${figmaAppearance}/Background/${emphasis}`;
    background = getVariableByName2(backgroundTokenName, tokenContext);
    if (background === null && attention === "high") {
      const a11yTokenName = `${figmaAppearance}/Background/${emphasis} a11y`;
      background = getVariableByName2(a11yTokenName, tokenContext);
    }
    if (background === null) {
      background = getVariableByName2(`Neutral/Background/${emphasis}`, tokenContext);
    }
  } else {
    background = "transparent";
  }
  const textLevelMap = {
    high: "High",
    medium: "Medium",
    low: "Low"
  };
  const textLevel = textLevelMap[attention];
  const hasBoldBackground = contained && attention === "high";
  const hasMediumColoredBackground = contained && attention === "medium" && appearance !== "neutral";
  let text = null;
  if (hasBoldBackground) {
    text = getVariableByName2(`${figmaAppearance}/Text/[On Bold] ${textLevel} Tinted`, tokenContext);
    if (text === null) {
      text = getVariableByName2(`${figmaAppearance}/Text/[On Bold] ${textLevel}`, tokenContext);
    }
    if (text === null) {
      text = getVariableByName2(`Neutral/Text/[On Bold] ${textLevel}`, tokenContext);
    }
  } else if (hasMediumColoredBackground) {
    text = getVariableByName2(`${figmaAppearance}/Text/High Tinted`, tokenContext);
    if (text === null) {
      text = getVariableByName2("Neutral/Text/High Tinted", tokenContext);
    }
  } else if (attention === "low") {
    text = getVariableByName2(`${figmaAppearance}/Text/Medium Tinted`, tokenContext);
    if (text === null) {
      text = getVariableByName2("Neutral/Text/Medium Tinted", tokenContext);
    }
  } else {
    text = getVariableByName2(`${figmaAppearance}/Text/${textLevel}`, tokenContext);
    if (text === null && contained) {
      text = getVariableByName2(`${figmaAppearance}/Text/${textLevel} Tinted`, tokenContext);
    }
    if (text === null) {
      text = getVariableByName2(`Neutral/Text/${textLevel}`, tokenContext);
    }
  }
  if (text === null) {
    const surfaceText = getVariableByName2("Neutral/Text/High", tokenContext);
    if (surfaceText !== null) {
      text = surfaceText;
    }
  }
  if (text === null) {
    console.error(
      "[Button.styles] TOKEN_UNRESOLVED: Text color token could not be resolved",
      { appearance, attention, textLevel, tokenContext }
    );
    text = colorMode === COLOR_MODE_MODES2.DARK ? "#FFFFFF" : "#000000";
  }
  let focus = getVariableByName2("Primary/Background/Bold", tokenContext);
  if (focus === null) {
    focus = getVariableByName2("Primary/Background/Bold a11ly", tokenContext);
  }
  if (focus === null) {
    console.error(
      "[Button.styles] TOKEN_UNRESOLVED: Focus ring color token could not be resolved",
      { tokenContext }
    );
    focus = "#0066FF";
  }
  return { background, text, focus };
}
function resolveTextDecoration(attention, contained, state) {
  if (!contained) {
    if (attention === "high") {
      return "underline";
    }
    if (attention === "medium" && (state === "hover" || state === "pressed" || state === "focus")) {
      return "underline";
    }
    if (attention === "low" && (state === "hover" || state === "pressed" || state === "focus")) {
      return "underline";
    }
  }
  return "none";
}
function determineButtonState(isPressed, isHovered, isFocused, isDisabled, isLoading) {
  if (isDisabled) return "disabled";
  if (isLoading) return "loading";
  if (isPressed) return "pressed";
  if (isHovered) return "hover";
  if (isFocused) return "focus";
  return "idle";
}

// src/components/Button/Button.tsx
import { Fragment, jsx as jsx3, jsxs } from "react/jsx-runtime";
var Button = forwardRef2(
  function Button2(props, forwardedRef) {
    const {
      // Design System props
      size = "M",
      attention = "medium",
      appearance = "auto",
      contained = true,
      condensed = false,
      single = false,
      fullWidth = false,
      loading = false,
      motionLevel: motionLevelProp,
      // Content slots (Slots 2, 3, 4 from blueprint)
      start,
      // Slot 2
      content,
      // Slot 3
      end,
      // Slot 4
      children,
      // Backward compatibility (maps to content)
      // HTML/React props
      className,
      style: customStyle,
      isDisabled,
      onPress,
      // Accessibility
      "aria-label": ariaLabel,
      ...restProps
    } = props;
    const slotContent = content ?? children;
    if (single && !ariaLabel && !loading) {
      console.error(
        "Button: Single buttons (single=true) require aria-label for accessibility"
      );
    }
    if (content && children) {
      console.warn(
        "Button: Both `content` and `children` props provided. Using `content`. Please use only `content` prop to follow the slot system."
      );
    }
    if (condensed && !contained) {
      console.warn(
        "Button: condensed property has no effect when contained=false"
      );
    }
    const buttonRef = forwardedRef || useRef(null);
    const [isHovered, setIsHovered] = useState3(false);
    const surfaceLevel = useSurfaceLevel();
    const dsContext = useDsContextOptional();
    const { motionLevel: detectedMotionLevel } = useMotion();
    const motionLevel = motionLevelProp ?? detectedMotionLevel;
    const disabled = isDisabled || loading;
    const hasBoldBackground = contained && attention === "high";
    const hasColoredBackground = contained && attention === "medium" && appearance !== "neutral";
    const { buttonProps, isPressed } = useButton(
      {
        ...restProps,
        isDisabled: disabled,
        onPress,
        "aria-label": ariaLabel
      },
      buttonRef
    );
    const { isFocusVisible, focusProps } = useFocusRing();
    const currentState = determineButtonState(
      isPressed,
      isHovered,
      false,
      // isFocused - using isFocusVisible instead
      disabled,
      loading
    );
    const resolvedStyles = resolveButtonStyles({
      size,
      attention,
      appearance,
      contained,
      condensed,
      single,
      fullWidth,
      state: currentState,
      surfaceLevel,
      disabled,
      loading,
      isFocusVisible,
      // Motion system integration
      motionLevel,
      isPressed,
      // Get platform, theme, colorMode, density from context (if available)
      platform: dsContext?.platform,
      theme: dsContext?.theme,
      colorMode: dsContext?.colorMode,
      density: dsContext?.density
    });
    const finalRootStyle = customStyle ? { ...resolvedStyles.root, ...customStyle } : resolvedStyles.root;
    const handleMouseEnter = () => setIsHovered(true);
    const handleMouseLeave = () => setIsHovered(false);
    const buttonTextColor = resolvedStyles.root.color;
    const renderSlot3Content = () => {
      if (loading) {
        return /* @__PURE__ */ jsx3("span", { style: resolvedStyles.content, children: /* @__PURE__ */ jsx3(
          Label,
          {
            as: "span",
            size,
            weight: "high",
            role: "status",
            "aria-live": "polite",
            style: { color: buttonTextColor },
            children: "Loading..."
          }
        ) });
      }
      if (single && typeof slotContent === "string") {
        return null;
      }
      return /* @__PURE__ */ jsx3("span", { style: resolvedStyles.content, children: /* @__PURE__ */ jsx3(Label, { as: "span", size, weight: "high", style: { color: buttonTextColor }, children: slotContent }) });
    };
    const buttonContent = /* @__PURE__ */ jsxs(Fragment, { children: [
      start && !loading && /* @__PURE__ */ jsx3("span", { style: resolvedStyles.start, children: start }),
      renderSlot3Content(),
      end && !loading && /* @__PURE__ */ jsx3("span", { style: resolvedStyles.end, children: end })
    ] });
    return /* @__PURE__ */ jsx3(
      "button",
      {
        ...mergeProps(buttonProps, focusProps),
        ref: buttonRef,
        className,
        style: finalRootStyle,
        onMouseEnter: handleMouseEnter,
        onMouseLeave: handleMouseLeave,
        "aria-busy": loading,
        type: "button",
        children: /* @__PURE__ */ jsx3(
          SurfaceProvider,
          {
            hasBoldBackground,
            hasColoredBackground,
            hasLowAttention: attention === "low",
            appearance,
            as: "span",
            style: { display: "contents" },
            children: buttonContent
          }
        )
      }
    );
  }
);
Button.displayName = "Button";
var Button_default = Button;
export {
  Button,
  Button_default as default,
  determineButtonState,
  resolveButtonStyles
};
