// @marcelinodzn/ds-tokens/dist/1-platform-D3C3RTVC.mjs
import "./chunk-4UEJOM6W.mjs";

// src/collections/1-platform.json
var schemaVersion = 1;
var collection = {
  id: "VariableCollectionId:2001:1427",
  name: "1. Platform",
  key: "183389ec7882730b91c46d2c5ba8b16a85c48171",
  hiddenFromPublishing: false,
  defaultModeId: "2001:8",
  modes: [
    {
      name: "Mobile (360)",
      modeId: "2001:8"
    },
    {
      name: "Tablet (768)",
      modeId: "19578:0"
    },
    {
      name: "Tablet Landscape (1024)",
      modeId: "19609:0"
    },
    {
      name: "Desktop (1440)",
      modeId: "2023:17"
    },
    {
      name: "Desktop (1920)",
      modeId: "24033:0"
    }
  ],
  remote: false,
  variableIds: [
    "VariableID:19059:71011",
    "VariableID:19059:71006",
    "VariableID:8650:18156",
    "VariableID:23169:86515",
    "VariableID:23169:86514",
    "VariableID:6046:8030",
    "VariableID:6046:8031",
    "VariableID:6046:8032",
    "VariableID:6046:8033",
    "VariableID:8875:19173",
    "VariableID:6046:8034",
    "VariableID:6046:8035",
    "VariableID:8803:517",
    "VariableID:8018:16757",
    "VariableID:8650:18158",
    "VariableID:8869:7655",
    "VariableID:8869:7657",
    "VariableID:8869:7659",
    "VariableID:8869:7661",
    "VariableID:8869:7663",
    "VariableID:8869:7665",
    "VariableID:8869:7667",
    "VariableID:8869:7668",
    "VariableID:8869:7669",
    "VariableID:8869:7670",
    "VariableID:8869:7671",
    "VariableID:9391:101780",
    "VariableID:9391:101778",
    "VariableID:2001:1433",
    "VariableID:6002:8130",
    "VariableID:2001:1432",
    "VariableID:2001:1431",
    "VariableID:2001:1430",
    "VariableID:8875:19165",
    "VariableID:2001:1428",
    "VariableID:6002:8131",
    "VariableID:6027:8780",
    "VariableID:6027:8781",
    "VariableID:14770:27283",
    "VariableID:6028:9160",
    "VariableID:14770:30709",
    "VariableID:6028:9164",
    "VariableID:6028:9163",
    "VariableID:6028:9162",
    "VariableID:6029:9165",
    "VariableID:8113:10184",
    "VariableID:15404:1729",
    "VariableID:2023:2079",
    "VariableID:6002:8136",
    "VariableID:6002:8137",
    "VariableID:6017:8555",
    "VariableID:21247:49705",
    "VariableID:12438:37540",
    "VariableID:6016:8509",
    "VariableID:6016:8510",
    "VariableID:6016:8511",
    "VariableID:6016:8514",
    "VariableID:6016:8515",
    "VariableID:6016:8516",
    "VariableID:8922:63456",
    "VariableID:8922:63455",
    "VariableID:6016:8519",
    "VariableID:6016:8520",
    "VariableID:6016:8521",
    "VariableID:6016:8522",
    "VariableID:8883:37947",
    "VariableID:8883:37946",
    "VariableID:6016:8523",
    "VariableID:6016:8524",
    "VariableID:6016:8525",
    "VariableID:6016:8526",
    "VariableID:6016:8527",
    "VariableID:6016:8528",
    "VariableID:6016:8529",
    "VariableID:6107:315",
    "VariableID:8309:3407",
    "VariableID:6001:11992",
    "VariableID:21088:273585",
    "VariableID:19283:96600",
    "VariableID:21088:273425",
    "VariableID:21088:273426",
    "VariableID:8309:3408",
    "VariableID:8309:3409",
    "VariableID:10872:3640",
    "VariableID:22915:35084",
    "VariableID:22915:35085",
    "VariableID:22922:2790",
    "VariableID:22925:21",
    "VariableID:23119:25",
    "VariableID:23119:2279",
    "VariableID:8922:63466",
    "VariableID:8922:63467",
    "VariableID:8922:63471",
    "VariableID:8922:63468",
    "VariableID:8922:63469",
    "VariableID:8922:63470",
    "VariableID:8922:63472",
    "VariableID:10597:798",
    "VariableID:10597:799",
    "VariableID:10597:1258"
  ],
  variables: [
    {
      id: "VariableID:19059:71011",
      name: "Spacing/5XL",
      description: "Only use to define como",
      key: "c47d0f1326b96a67a1c17c6e0cf78e09f2e60717",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7882"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8002"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1049"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146762"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12304"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:19059:71006",
      name: "Spacing/4XL",
      description: "Only use to define como",
      key: "a5e3ea7eb38b847ba20454f60e8152f9e98b71f2",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7878"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8004"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1051"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146764"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12306"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8650:18156",
      name: "Dimension/5XL",
      description: "",
      key: "06933511af7e8d430e708edc8fabd1cc5b0d2d69",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7882"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8002"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1049"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146762"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12304"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:23169:86515",
      name: "Shape/2XL",
      description: "",
      key: "72c91b0b45923060ceaa4eb0229a637aa4a652e8",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7865"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8008"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1055"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146768"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12310"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:23169:86514",
      name: "Shape/XL",
      description: "",
      key: "51871bf0cab0f3b9bc46a35eb08f6cf96e8808fb",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7867"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8010"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1057"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146770"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12312"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6046:8030",
      name: "Shape/L",
      description: "",
      key: "1a272f556d9f3dd84d8ba53abe09fcf6100743b1",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7869"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8012"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1059"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146772"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12314"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6046:8031",
      name: "Shape/M",
      description: "",
      key: "4e5927c5f3f47b2417a6a56ca3e95fd5a82efb29",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7862"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8014"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1061"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146774"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12316"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6046:8032",
      name: "Shape/S",
      description: "",
      key: "8d251490182a4b8d48080e3cf70d405e5682c91c",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7872"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8016"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1063"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146776"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12318"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6046:8033",
      name: "Shape/XS",
      description: "",
      key: "7c2ea4001ae60183fdf6868775d1cbbd04fe16cf",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7874"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8018"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1065"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146778"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12320"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8875:19173",
      name: "Shape/2XS",
      description: "",
      key: "92032948a49eeb91e3b9c058dc27b3f9e9cbdef2",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7875"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8019"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1066"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146779"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12321"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6046:8034",
      name: "Shape/3XS",
      description: "",
      key: "08b4f4368241a8fa462e7cc112ba43621f62e479",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7876"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8020"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1067"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146780"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12322"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6046:8035",
      name: "Shape/4XS",
      description: "",
      key: "7e8faddd83fb9ca9d2b8dfa9bef9e33891db9d8d",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7884"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8021"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1068"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146781"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12323"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8803:517",
      name: "Shape/00",
      description: "",
      key: "e22bd83148223ce2fee770c9e98bd95899a9d39d",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7885"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8022"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1069"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146782"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12324"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8018:16757",
      name: "Shape/Pill",
      description: "",
      key: "8a3b807f79485e4650ec6c4e7b6926c02357486f",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 999,
        "2023:17": 999,
        "19578:0": 999,
        "19609:0": 999,
        "24033:0": 999
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8650:18158",
      name: "Dimension/4XL",
      description: "",
      key: "22baf01d646d3d53a693c1c1783a2fa3400fc417",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7878"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8004"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1051"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146764"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12306"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7655",
      name: "Dimension/3XL",
      description: "",
      key: "67efe117e9a8780296e89c3c15a10c1495594069",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7863"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8006"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1053"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146766"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12308"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7657",
      name: "Dimension/2XL",
      description: "",
      key: "6007d08c310b92b531dc711710a3b0dd3925f234",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7865"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8008"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1055"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146768"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12310"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7659",
      name: "Dimension/XL",
      description: "",
      key: "d62ebc676ed5bfb06470b4bc5d0d86c11819797f",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7867"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8010"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1057"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146770"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12312"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7661",
      name: "Dimension/L",
      description: "",
      key: "41b4c22737c1f7cce5347abad1e7246615227e92",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7869"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8012"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1059"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146772"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12314"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7663",
      name: "Dimension/M",
      description: "",
      key: "7f241fb8ceaf6830f60b50f279d2c5d4b39a4de6",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7862"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8014"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1061"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146774"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12316"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7665",
      name: "Dimension/S",
      description: "",
      key: "a282fbe66ddc8c37b23b4962e9381ae0d99b86fb",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7872"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8016"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1063"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146776"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12318"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7667",
      name: "Dimension/XS",
      description: "",
      key: "8e4d1ea5d26c3345d1c65a1c9856ec32b1de77c5",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7874"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8018"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1065"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146778"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12320"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7668",
      name: "Dimension/2XS",
      description: "",
      key: "a7c0d9de8a2c3f65f1b133faa9d2bf388827d367",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7875"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8019"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1066"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146779"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12321"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7669",
      name: "Dimension/3XS",
      description: "",
      key: "04a5d3e37a4526edf0db296abd9984c1196ab389",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7876"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8020"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1067"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146780"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12322"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7670",
      name: "Dimension/4XS",
      description: "",
      key: "f42f2e6655177ff1d28253fd9372dc3447019255",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7884"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8021"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1068"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146781"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12323"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8869:7671",
      name: "Dimension/00",
      description: "",
      key: "d6aafaf14200ffbf2a49910175bfea5e67b87a0f",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7885"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8022"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1069"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146782"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12324"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "FONT_VARIATIONS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:9391:101780",
      name: "Spacing/3XL",
      description: "Only use to define como",
      key: "118e4e151352cb060c34232cf9137f99a80a2f28",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7863"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8006"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1053"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146766"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12308"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:9391:101778",
      name: "Spacing/2XL",
      description: "Only use to define como",
      key: "3abf70fdede70ddf6b9c86fcce5aac1c61330420",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7865"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8008"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1055"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146768"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12310"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:2001:1433",
      name: "Spacing/XL",
      description: "",
      key: "e5d7f2f52182fbc1f525bd308ceba47abe25e416",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7867"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8010"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1057"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146770"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12312"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6002:8130",
      name: "Spacing/L",
      description: "",
      key: "f4478958914f7a20c51f3853352e9b9ee62c469d",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7869"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8012"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1059"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146772"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12314"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:2001:1432",
      name: "Spacing/M",
      description: "",
      key: "28a53ca87a0b526ed98e7a97347459231fe6bddb",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7862"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8014"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1061"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146774"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12316"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:2001:1431",
      name: "Spacing/S",
      description: "",
      key: "16da2e3fac2e7e7a4ce03ffc9e70ccfd753b4ec6",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7872"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8016"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1063"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146776"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12318"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:2001:1430",
      name: "Spacing/XS",
      description: "",
      key: "4ef96bb48c04a9993b7f9cf891de1b41cd217d5b",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7874"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8018"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1065"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146778"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12320"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8875:19165",
      name: "Spacing/2XS",
      description: "",
      key: "3b25751fff5df76f34032759b1b09af549742748",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7875"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8019"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1066"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146779"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12321"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:2001:1428",
      name: "Spacing/3XS",
      description: "",
      key: "99797f76ad77feacf5b068be6c90174e0288164c",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7876"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8020"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1067"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146780"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12322"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6002:8131",
      name: "Spacing/4XS",
      description: "",
      key: "37cca8f21eb46d9c813d6c78ab766c1e38aea433",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7884"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8021"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1068"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146781"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12323"
        }
      },
      scopes: [
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6027:8780",
      name: "Typography/Font Weight/Display",
      description: "",
      key: "16ff727633eb2fc6027ba88359b017b79ee28053",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 900,
        "2023:17": 900,
        "19578:0": 900,
        "19609:0": 900,
        "24033:0": 900
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6027:8781",
      name: "Typography/Font Weight/Headline High",
      description: "",
      key: "ef53572a5b867e61f5215896a80673c703a6d6f3",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 900,
        "2023:17": 900,
        "19578:0": 900,
        "19609:0": 900,
        "24033:0": 900
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:14770:27283",
      name: "Typography/Font Weight/Headline Medium",
      description: "",
      key: "01cf478d9eb0008390cf7536e438f5d04cd7d256",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 850,
        "2023:17": 850,
        "19578:0": 850,
        "19609:0": 850,
        "24033:0": 850
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6028:9160",
      name: "Typography/Font Weight/Title High",
      description: "",
      key: "8690c38756ca93d0c915a928c1f99ddd9a7b9d49",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 800,
        "2023:17": 800,
        "19578:0": 800,
        "19609:0": 800,
        "24033:0": 800
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:14770:30709",
      name: "Typography/Font Weight/Title Medium",
      description: "",
      key: "b1d37c629947b335f0a4b3063eaa77f13b088818",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 750,
        "2023:17": 750,
        "19578:0": 750,
        "19609:0": 750,
        "24033:0": 750
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6028:9164",
      name: "Typography/Font Weight/Body High",
      description: "",
      key: "af81bb096dcdbbec7df9ac619c91eddf9c7d00c7",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 700,
        "2023:17": 700,
        "19578:0": 700,
        "19609:0": 700,
        "24033:0": 700
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6028:9163",
      name: "Typography/Font Weight/Body Medium",
      description: "",
      key: "94d23898f7c997b2cd7f5ed6203dca48573f0baa",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 500,
        "2023:17": 500,
        "19578:0": 500,
        "19609:0": 500,
        "24033:0": 500
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6028:9162",
      name: "Typography/Font Weight/Body Low (Regular)",
      description: "",
      key: "505440bc8d3d7225372a3106af1391eeaf1e26e6",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 400,
        "2023:17": 400,
        "19578:0": 400,
        "19609:0": 400,
        "24033:0": 400
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6029:9165",
      name: "Typography/Font Weight/Label High",
      description: "",
      key: "9ca13f91ba1786714293b23394442ec0b5b0722e",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 700,
        "2023:17": 700,
        "19578:0": 700,
        "19609:0": 700,
        "24033:0": 700
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8113:10184",
      name: "Typography/Font Weight/Label Medium",
      description: "",
      key: "51b71af2ab0b0494ab06a552c9123abe17a9c7b3",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 500,
        "2023:17": 500,
        "19578:0": 500,
        "19609:0": 500,
        "24033:0": 500
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:15404:1729",
      name: "Typography/Font Weight/Label Low",
      description: "",
      key: "5517be80660bd7678bac1325fd8fe522c5f08295",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 400,
        "2023:17": 400,
        "19578:0": 400,
        "19609:0": 400,
        "24033:0": 400
      },
      scopes: [
        "FONT_WEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:2023:2079",
      name: "Typography/Size/Display/L",
      description: "",
      key: "7302b50750c46419cf44b7c4d4e69bf758e577d2",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7864"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8007"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1054"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146767"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12309"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6002:8136",
      name: "Typography/Size/Display/M",
      description: "",
      key: "edf0a084eba1ee4f236e3b193887fe5cabbdd838",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7865"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8008"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1055"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146768"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12310"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6002:8137",
      name: "Typography/Size/Display/S",
      description: "",
      key: "f74715f6fcd4c79589874b0e9c4744e4dfffd89a",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7866"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8009"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1056"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146769"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12311"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6017:8555",
      name: "Typography/Font Family",
      description: "",
      key: "0634b0bcccda44e692507b0b54805395563359d5",
      resolvedType: "STRING",
      valuesByMode: {
        "2001:8": "JioType Var",
        "2023:17": "JioType Var",
        "19578:0": "JioType Var",
        "19609:0": "JioType Var",
        "24033:0": "JioType Var"
      },
      scopes: [
        "ALL_SCOPES"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:21247:49705",
      name: "Typography/Font Family [code]",
      description: "",
      key: "8ff2458209950097c6edabf9f768c67c66e26c6c",
      resolvedType: "STRING",
      valuesByMode: {
        "2001:8": "JetBrains Mono",
        "2023:17": "JetBrains Mono",
        "19578:0": "JetBrains Mono",
        "19609:0": "JetBrains Mono",
        "24033:0": "JetBrains Mono"
      },
      scopes: [
        "ALL_SCOPES"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:12438:37540",
      name: "Typography/Display+Headline",
      description: "",
      key: "b5f322621d60555c459d5934934bf9e3fb8bc01d",
      resolvedType: "STRING",
      valuesByMode: {
        "2001:8": "Black",
        "2023:17": "Black",
        "19578:0": "Black",
        "19609:0": "Black",
        "24033:0": "Black"
      },
      scopes: [
        "ALL_SCOPES"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8509",
      name: "Typography/Size/Headline/L",
      description: "",
      key: "060d8974445e9470ae9e05332ce3fc1d678035c1",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7867"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8010"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1057"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146770"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12312"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8510",
      name: "Typography/Size/Headline/M",
      description: "",
      key: "bc696d47f1befc2be6bcb5d326973a48495dc9b7",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7869"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8012"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1059"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146772"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12314"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8511",
      name: "Typography/Size/Headline/S",
      description: "",
      key: "9578e8a675c114a38e72e5ec21f72acee7b85e4a",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7862"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8014"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1061"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146774"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12316"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8514",
      name: "Typography/Size/Title/L",
      description: "",
      key: "e3b6e5dffaa2f7b9dabf72f878d0fda1dfb99449",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7869"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8012"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1059"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146772"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12314"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8515",
      name: "Typography/Size/Title/M",
      description: "",
      key: "088f816d1cbfdeaebe61bc32b337144dde04b522",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7862"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8014"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1061"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146774"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12316"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8516",
      name: "Typography/Size/Title/S",
      description: "",
      key: "8e0d48612c5b9dbab8cd3ff6e1e050e626b3f593",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7872"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8016"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1063"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146776"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12318"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8922:63456",
      name: "Typography/Size/Body/2XL",
      description: "",
      key: "120e35908a0a9dd391b1a3bc591a12644598625b",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7868"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8011"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1058"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146771"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12313"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8922:63455",
      name: "Typography/Size/Body/XL",
      description: "",
      key: "f1c9bfaa9617c47c368bdff199388d3dc8d59e3d",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7869"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8012"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1059"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146772"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12314"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8519",
      name: "Typography/Size/Body/L",
      description: "",
      key: "9a8e6470a0970d0e63b8b4a79aacb287a32dad03",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7870"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8013"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1060"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146773"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12315"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8520",
      name: "Typography/Size/Body/M",
      description: "",
      key: "618ed1238ea8a4075fce8c8ff6d6be209ad390a3",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7862"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8014"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1061"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146774"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12316"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8521",
      name: "Typography/Size/Body/S",
      description: "",
      key: "c752b34a60fa25dd91d5ecc99abdb6a309c1360c",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7871"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8015"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1062"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146775"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12317"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8522",
      name: "Typography/Size/Body/XS",
      description: "",
      key: "127af8033d20715fa10f372a7d69883b8971beb4",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7872"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8016"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1063"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146776"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12318"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8883:37947",
      name: "Typography/Size/Label/2XL",
      description: "",
      key: "c8aee163fc1abb15d12becd6565bca34c29c6818",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7868"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8011"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1058"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146771"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12313"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: true,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8883:37946",
      name: "Typography/Size/Label/XL",
      description: "",
      key: "a6a16a3931da897bce365aa34c4a8e617fcd9de0",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7869"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8012"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1059"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146772"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12314"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: true,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8523",
      name: "Typography/Size/Body/2XS",
      description: "",
      key: "c66d121a1b9102c1a4fda9400c3c8a72ec2a8bd9",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7873"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8017"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1064"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146777"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12319"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8524",
      name: "Typography/Size/Label/L",
      description: "",
      key: "7c723a8d864f778edf315b773415168f5a149e85",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7870"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8013"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1060"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146773"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12315"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8525",
      name: "Typography/Size/Label/M",
      description: "",
      key: "48438d971a538bd21ef9e546ce1b73cb81a7d9a3",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7862"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8014"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1061"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146774"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12316"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8526",
      name: "Typography/Size/Label/S",
      description: "",
      key: "a891cf35c198478da2860b9eca6bc272bb8060fc",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7871"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8015"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1062"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146775"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12317"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8527",
      name: "Typography/Size/Label/XS",
      description: "",
      key: "046072ce55ca6558d2c7ec52b998336309b4207c",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7872"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8016"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1063"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146776"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12318"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8528",
      name: "Typography/Size/Label/2XS",
      description: "",
      key: "f8245c2c848cc64c251eee75b1015756be59491f",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7873"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8017"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1064"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146777"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12319"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6016:8529",
      name: "Typography/Size/Label/3XS",
      description: "",
      key: "6ff91e94d87c72ac7ff0e8f48e462c8d246e9e1d",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:7874"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6002:8018"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19574:1065"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:146778"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12320"
        }
      },
      scopes: [
        "FONT_SIZE"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6107:315",
      name: "Platform",
      description: "",
      key: "e393bf401043de5be91c8cc168c4500544f67b2e",
      resolvedType: "STRING",
      valuesByMode: {
        "2001:8": "Mobile (360)",
        "2023:17": "Desktop (1440)",
        "19578:0": "Tablet (768)",
        "19609:0": "Tablet (1024)",
        "24033:0": "Desktop (1920)"
      },
      scopes: [
        "ALL_SCOPES"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8309:3407",
      name: "Grid/margin",
      description: "",
      key: "153d574b13e99bc0bb67540c3eeea38f5ab5d79f",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8305:3294"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8309:3404"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19576:1351"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:182150"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12328"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:6001:11992",
      name: "Screen size/Screen width",
      description: "",
      key: "794b2b06442fbea0f084db8c6760c597b6d30fd8",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 360,
        "2023:17": 1440,
        "19578:0": 768,
        "19609:0": 1024,
        "24033:0": 1920
      },
      scopes: [
        "WIDTH_HEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:21088:273585",
      name: "Screen size/Screen height",
      description: "",
      key: "f5221b642ba0157a2837f091f14798d192d1d90e",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 800,
        "2023:17": 900,
        "19578:0": 1024,
        "19609:0": 768,
        "24033:0": 1080
      },
      scopes: [
        "WIDTH_HEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:19283:96600",
      name: "Container/100-100-100 (Full width)",
      description: "",
      key: "577183d300823f143aaacd67e9992c2d960db9a2",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19281:75598"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19281:96598"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19576:1353"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:195147"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24038:175346"
        }
      },
      scopes: [
        "WIDTH_HEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:21088:273425",
      name: "Container/100-50-50",
      description: "",
      key: "fce4304d9d35e8079f85488c532690f7e9e43283",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21088:273300"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21088:273408"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21088:273406"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21088:273407"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24038:175347"
        }
      },
      scopes: [
        "WIDTH_HEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:21088:273426",
      name: "Container/100-50-33",
      description: "",
      key: "56d10d9119ecbaac6d980d5436c01856c0dc0410",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21088:273301"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21088:273411"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21088:273409"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21088:273410"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24038:175348"
        }
      },
      scopes: [
        "WIDTH_HEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8309:3408",
      name: "Grid/gutter",
      description: "",
      key: "1b1212e951a57118bbce41f74a27838065512d03",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8305:3295"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8309:3405"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19576:1352"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:19609:182151"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12329"
        }
      },
      scopes: [
        "WIDTH_HEIGHT",
        "GAP"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8309:3409",
      name: "Grid/columns",
      description: "",
      key: "7c6f5c3fd5cf3de9e69f2a8ea2c058e3ecc58f3d",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 4,
        "2023:17": 12,
        "19578:0": 8,
        "19609:0": 8,
        "24033:0": 12
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:10872:3640",
      name: "Stroke/None",
      description: "",
      key: "21477b7456f942ccb4ba8e26069ac7395f21fefe",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 0,
        "2023:17": 0,
        "19578:0": 0,
        "19609:0": 0,
        "24033:0": 0
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:22915:35084",
      name: "Component Tokens/ChatInput/OuterRadius",
      description: "",
      key: "4bfb20fa6c7c7c19538d0321df627646b09988a5",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22914:14272"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22915:34987"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22915:34983"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22915:34985"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12330"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:22915:35085",
      name: "Component Tokens/ChatInput/InnerRadiusS",
      description: "",
      key: "b6fc264eee9815221c9a09a20dbff0f9b7e3218c",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22915:34982"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22915:34988"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22915:34984"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22915:34986"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12332"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:22922:2790",
      name: "Component Tokens/ChatInput/InnerRadiusM",
      description: "",
      key: "eada47a22b362e20c642c7129b84a3307537f13f",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22922:20"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22922:2722"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22922:21"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:22922:2721"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12331"
        }
      },
      scopes: [
        "CORNER_RADIUS"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:22925:21",
      name: "Component Tokens/ChatInput/MaxWidth",
      description: "",
      key: "0e522d3a01a6de5e9beb7054e1ca9da22d40fc75",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 800,
        "2023:17": 800,
        "19578:0": 800,
        "19609:0": 800,
        "24033:0": 800
      },
      scopes: [
        "WIDTH_HEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:23119:25",
      name: "Component Tokens/ChatInput/AttachmentHeight",
      description: "",
      key: "9c89038a3077b08812c6907962693db8f6796ace",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:23119:20"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:23119:23"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:23119:21"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:23119:22"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12333"
        }
      },
      scopes: [
        "WIDTH_HEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:23119:2279",
      name: "Component Tokens/ChatInput/AttachmentWidth",
      description: "",
      key: "a7d68fa79255759dc8ef5e3f3100690b8184ae08",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:23119:2274"
        },
        "2023:17": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:23119:2278"
        },
        "19578:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:23119:2276"
        },
        "19609:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:23119:2277"
        },
        "24033:0": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:24033:12334"
        }
      },
      scopes: [
        "WIDTH_HEIGHT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8922:63466",
      name: "Stroke/S",
      description: "",
      key: "134bc6af2c8de9e074ac41ca7834bfcb56fee302",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 0.5,
        "2023:17": 0.5,
        "19578:0": 0.5,
        "19609:0": 0.5,
        "24033:0": 0.5
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8922:63467",
      name: "Stroke/M",
      description: "",
      key: "a20a091c694d0286e2f2dea270b2c9e236f5288b",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 1,
        "2023:17": 1,
        "19578:0": 1,
        "19609:0": 1,
        "24033:0": 1
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8922:63471",
      name: "Stroke/L",
      description: "",
      key: "07143e0151d825ffb7550279a226fe1999924144",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 1.5,
        "2023:17": 1.5,
        "19578:0": 1.5,
        "19609:0": 1.5,
        "24033:0": 1.5
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8922:63468",
      name: "Stroke/XL",
      description: "",
      key: "235e65bab4e23c892016c0add7f98f18a832c820",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 2,
        "2023:17": 2,
        "19578:0": 2,
        "19609:0": 2,
        "24033:0": 2
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8922:63469",
      name: "Stroke/2XL",
      description: "",
      key: "8dbbeaafe077a067e41a7a335fc2f083dac71d1e",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 3,
        "2023:17": 3,
        "19578:0": 3,
        "19609:0": 3,
        "24033:0": 3
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8922:63470",
      name: "Stroke/3XL",
      description: "",
      key: "dd5eb9e4cf0e435146a29ea4c046497599bd025c",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 5,
        "2023:17": 5,
        "19578:0": 5,
        "19609:0": 5,
        "24033:0": 5
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:8922:63472",
      name: "Stroke/4XL",
      description: "",
      key: "50f541bea43948be5823a5b5a076be6a34289d39",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 8,
        "2023:17": 8,
        "19578:0": 8,
        "19609:0": 8,
        "24033:0": 8
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:10597:798",
      name: "Stroke/5XL",
      description: "",
      key: "b3977046646f474a0919e30a88a65ab7d2472f7d",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 11,
        "2023:17": 11,
        "19578:0": 11,
        "19609:0": 11,
        "24033:0": 11
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:10597:799",
      name: "Stroke/6XL",
      description: "",
      key: "44458820e9046585193d4bad5e0eeae6cee3a346",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 17,
        "2023:17": 17,
        "19578:0": 17,
        "19609:0": 17,
        "24033:0": 17
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    },
    {
      id: "VariableID:10597:1258",
      name: "Stroke/7XL",
      description: "",
      key: "5ea649ccc45774f0c9d37cc8b65342a52adbc2b6",
      resolvedType: "FLOAT",
      valuesByMode: {
        "2001:8": 26,
        "2023:17": 26,
        "19578:0": 26,
        "19609:0": 26,
        "24033:0": 26
      },
      scopes: [
        "WIDTH_HEIGHT",
        "STROKE_FLOAT",
        "EFFECT_FLOAT"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:2001:1427"
    }
  ]
};
var platform_default = {
  schemaVersion,
  collection
};
export {
  collection,
  platform_default as default,
  schemaVersion
};
