// @marcelinodzn/ds-tokens/dist/5-appearance-QDLPJB4J.mjs
import "./chunk-4UEJOM6W.mjs";

// src/collections/5-appearance.json
var schemaVersion = 1;
var collection = {
  id: "VariableCollectionId:16178:28095",
  name: "5. \u{E0003}Appearance",
  key: "a7836a9203871299dfe27061ca103d3b1575cebf",
  hiddenFromPublishing: false,
  defaultModeId: "16178:1",
  modes: [
    {
      name: "\u2193Brand Appearance",
      modeId: "16178:1"
    },
    {
      name: "\u2193Semantic",
      modeId: "16178:2"
    }
  ],
  remote: false,
  variableIds: [
    "VariableID:16178:28096",
    "VariableID:16178:28097",
    "VariableID:16178:28098",
    "VariableID:16178:28099",
    "VariableID:16178:28100",
    "VariableID:16178:28101",
    "VariableID:16178:28102",
    "VariableID:16178:28103",
    "VariableID:16178:28104",
    "VariableID:16178:28105",
    "VariableID:16178:28106",
    "VariableID:16178:28107",
    "VariableID:16178:28108",
    "VariableID:16178:28109",
    "VariableID:16178:28110",
    "VariableID:16178:28111",
    "VariableID:16485:94824",
    "VariableID:16485:94825",
    "VariableID:16178:28112",
    "VariableID:16485:87683",
    "VariableID:16178:28113",
    "VariableID:16178:28114",
    "VariableID:21360:2908",
    "VariableID:21360:2909",
    "VariableID:21360:2910",
    "VariableID:21360:2911",
    "VariableID:21360:2912",
    "VariableID:21360:2913",
    "VariableID:21360:2914",
    "VariableID:21360:2915",
    "VariableID:21360:2916",
    "VariableID:21360:2917",
    "VariableID:21360:2918",
    "VariableID:16178:28115",
    "VariableID:16178:28116",
    "VariableID:16178:28117",
    "VariableID:16178:28118",
    "VariableID:16178:28119",
    "VariableID:16513:56595",
    "VariableID:16513:56596",
    "VariableID:16178:28120",
    "VariableID:16513:33931",
    "VariableID:16178:28121",
    "VariableID:16178:28122",
    "VariableID:16178:28123",
    "VariableID:16178:28124",
    "VariableID:16178:28125",
    "VariableID:16178:28126",
    "VariableID:16178:28127",
    "VariableID:16178:28128",
    "VariableID:16178:28129",
    "VariableID:16178:28130",
    "VariableID:16178:28131",
    "VariableID:16178:28132",
    "VariableID:16178:28133"
  ],
  variables: [
    {
      id: "VariableID:16178:28096",
      name: "Surface/Ghost",
      description: "",
      key: "6702ef43bec18a8e064fd5fd8b1b06638a0897c2",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6070:133"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6074:20"
        }
      },
      scopes: [
        "FRAME_FILL",
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28097",
      name: "Surface/Minimal",
      description: "",
      key: "61a91090a8252815005cdf1a68b046e94ea505ca",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6070:134"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6074:21"
        }
      },
      scopes: [
        "FRAME_FILL",
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28098",
      name: "Surface/Subtle",
      description: "",
      key: "6c2537fb308a95f578e1d2265e30a0170fd72682",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6070:135"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6074:22"
        }
      },
      scopes: [
        "FRAME_FILL",
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28099",
      name: "Surface/Moderate",
      description: "",
      key: "896b5956023a8fb91bf057d88859bb4bd63b67a0",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6070:136"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6074:23"
        }
      },
      scopes: [
        "FRAME_FILL",
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28100",
      name: "Surface/Bold",
      description: "",
      key: "2c355df1e6d53d3cefa3a5fe91854d4e41ee51b5",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6070:137"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6073:10"
        }
      },
      scopes: [
        "FRAME_FILL",
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28101",
      name: "Surface/Heavy",
      description: "",
      key: "6880aa12774d887434f10251a29bb10231aa5b2c",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6070:138"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6074:24"
        }
      },
      scopes: [
        "FRAME_FILL",
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28102",
      name: "Surface/Contrasting",
      description: "",
      key: "3c1f5730f601bb86d6e5b17ef288ec04fa2081a4",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6070:139"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6074:25"
        }
      },
      scopes: [
        "FRAME_FILL",
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28103",
      name: "Surface/Elevated Low",
      description: "",
      key: "5bd5797ae0ebdaf5249b95e14653816711b12062",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:12354:270"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:13241:1494"
        }
      },
      scopes: [
        "FRAME_FILL",
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28104",
      name: "Surface/Elevated Medium",
      description: "",
      key: "e9f69268424e4333c30f5c9997213ff7d30cdb56",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:13241:1493"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:13241:1495"
        }
      },
      scopes: [
        "FRAME_FILL",
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28105",
      name: "Surface/Elevated High",
      description: "",
      key: "ea92defb598ad001e0fed04568682be9676413c5",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:13396:39293"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:13396:39294"
        }
      },
      scopes: [
        "FRAME_FILL",
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28106",
      name: "Text/Low",
      description: "",
      key: "b9b21a1e3d0a6850b8bb1f3c1f129e89bf36e71c",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6110:342"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8104:1034"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28107",
      name: "Text/Medium",
      description: "",
      key: "0b9836956205d555aba13068058ba8f0ef8c194e",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6110:343"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8104:1035"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28108",
      name: "Text/High",
      description: "",
      key: "3fc2b1d0a808862bca77ac55644d82d2b8241abb",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6110:344"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8104:1036"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28109",
      name: "Text/Low Tinted",
      description: "",
      key: "84b2b45bd7819b5f2acae5a93dd068240147d811",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15997:178489"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15999:513"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28110",
      name: "Text/Medium Tinted",
      description: "",
      key: "46adeff1350319a481ecd72fa6e7c32186d45cdb",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6157:1366"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8104:1033"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28111",
      name: "Text/High Tinted",
      description: "",
      key: "232bba28467dda1771f90185126f9221a12cbb3e",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15997:178490"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15999:514"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16485:94824",
      name: "Text/[On Bold] Medium",
      description: "",
      key: "83613f0ab2316f8ce8e3d21dac0297ceb63d4710",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:85584"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:91614"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16485:94825",
      name: "Text/[On Bold] High",
      description: "",
      key: "6539675ce4e1c9a389490e41ea735456a3289f31",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:85585"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:91615"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28112",
      name: "Text/[On Bold] Medium Tinted",
      description: "",
      key: "406e3645749f07f7af4aae5aa9290698ec14f8c3",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6070:201"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6073:11"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16485:87683",
      name: "Text/[On Bold] High Tinted",
      description: "",
      key: "1172edccf3810c8f779fff1903af4ab4db87e431",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:82050"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:87684"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28113",
      name: "Text/[On Contrasting]",
      description: "",
      key: "c01220328a2f7f1ec4fe0bca2b8d70057635354d",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:9772:60089"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:9772:60090"
        }
      },
      scopes: [
        "TEXT_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28114",
      name: "Icon/Low",
      description: "",
      key: "8d45ebaa5d7457093845fdd735dba041e3c85847",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8976:72093"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8976:72098"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2908",
      name: "Icon [isolated]/Low",
      description: "",
      key: "3ea441ccf2c71887f34a693a1c68c79d8e9f3260",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2919"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2930"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2909",
      name: "Icon [isolated]/Medium",
      description: "",
      key: "91148c4a3e3b1f55b978d89f1a09ad9356a65ac6",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2920"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2931"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2910",
      name: "Icon [isolated]/High",
      description: "",
      key: "04c6eb0195fef6d405c532a9aa7b3ea3cfd5c7a8",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2921"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2932"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2911",
      name: "Icon [isolated]/Low Tinted",
      description: "",
      key: "40044748e639c2d90e2fdb39d85e11a734d3024b",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2922"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2933"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2912",
      name: "Icon [isolated]/Medium Tinted",
      description: "",
      key: "715d80d7182fe1096c8c4c01b86a716ce786d807",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2923"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2934"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2913",
      name: "Icon [isolated]/High Tinted",
      description: "",
      key: "037b21a9a01199023610134aca88f5cb2d96b98a",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2924"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2935"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2914",
      name: "Icon [isolated]/[On Bold] Medium",
      description: "",
      key: "437f3e99f36ea86adb1e29d0015c91d4f089e6e8",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2925"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2936"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2915",
      name: "Icon [isolated]/[On Bold] High",
      description: "",
      key: "5a5e6f1d00b3c76f21add68efcee080dcf1f049b",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2926"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2937"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2916",
      name: "Icon [isolated]/[On Bold] Medium Tinted",
      description: "",
      key: "bf207e89aceff5c055e178cf40d4bd1eb0b70592",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2927"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2938"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2917",
      name: "Icon [isolated]/[On Bold] High Tinted",
      description: "",
      key: "de56cac289550a45d107c31e4a1e9cd894bd758c",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2928"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2939"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:21360:2918",
      name: "Icon [isolated]/[On Contrasting]",
      description: "",
      key: "ae2f8ff5efc96cc47fd673b3f445dd7614837760",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21360:2929"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:21361:2940"
        }
      },
      scopes: [],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28115",
      name: "Icon/Medium",
      description: "",
      key: "00f6a20685764d3b85ee45b9255298d1e23ef49a",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8976:72094"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8976:72099"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28116",
      name: "Icon/High",
      description: "",
      key: "c05c3cd5bcff2f2a8c3d18e58a1761a9c353e4a5",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8976:72096"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8976:72101"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28117",
      name: "Icon/Low Tinted",
      description: "",
      key: "81f4b5edb6fbdc12acb77cdf90bbb192216cf314",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15997:178499"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15999:7179"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28118",
      name: "Icon/Medium Tinted",
      description: "",
      key: "8fb3087f5e7f3c8a306b0cb221d4fba06cd37d7f",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:9324:26875"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:9324:26877"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28119",
      name: "Icon/High Tinted",
      description: "",
      key: "fcf57e7ab4adfe7c64ee957bc782c39ac9f70ec1",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15997:178500"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15999:7180"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16513:56595",
      name: "Icon/[On Bold] Medium",
      description: "",
      key: "280b2a8400a7e67d5ebb26f756ce151481892fd6",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:87657"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:94799"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16513:56596",
      name: "Icon/[On Bold] High",
      description: "",
      key: "7d2f188a0ccb0ef547f5b7f7518d6adb6b2acfc0",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:87658"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:94800"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28120",
      name: "Icon/[On Bold] Medium Tinted",
      description: "",
      key: "7e53e689213eeadd8216e8fe222e7b497d5c68c9",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8976:72097"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8976:72102"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16513:33931",
      name: "Icon/[On Bold] High Tinted",
      description: "",
      key: "375aca58d6c493c5ed4c69bd945f849ebd0a63af",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:87656"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:16485:94798"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28121",
      name: "Icon/[On Contrasting]",
      description: "",
      key: "9f9fe8b79afe222890a7d9e75948f6fca0e0da1d",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:12489:76041"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:12491:222289"
        }
      },
      scopes: [
        "SHAPE_FILL"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28122",
      name: "Stroke/Subtle",
      description: "",
      key: "a2d0641ee93b876bfe36e322052ffda83b79930e",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6127:1533"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8104:1037"
        }
      },
      scopes: [
        "STROKE_COLOR"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28123",
      name: "Stroke/Moderate",
      description: "",
      key: "855526970b156d0dfef127c3128e3f898337414c",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6127:1534"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8104:1042"
        }
      },
      scopes: [
        "STROKE_COLOR"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28124",
      name: "Stroke/Bold",
      description: "",
      key: "77e64ea2ee2b05487973b3c0eec66f61de1cbbf1",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15997:178503"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15999:23560"
        }
      },
      scopes: [
        "STROKE_COLOR"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28125",
      name: "Stroke/Heavy",
      description: "",
      key: "367b1ccb45b0d74fc291ec2f5a28ab58a233fd47",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6157:873"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8104:1043"
        }
      },
      scopes: [
        "STROKE_COLOR"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28126",
      name: "Stroke/Contrasting",
      description: "",
      key: "cc220c4c2e5bb30855b3c370e2f3ebe747104c30",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6127:1535"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8104:1044"
        }
      },
      scopes: [
        "STROKE_COLOR"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28127",
      name: "Stroke/Subtle Tinted",
      description: "",
      key: "fb24812e1a3aa24a523b67e46998f90620fd0f1f",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15997:178501"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15999:23558"
        }
      },
      scopes: [
        "STROKE_COLOR"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28128",
      name: "Stroke/Moderate Tinted",
      description: "",
      key: "da0cc32a4d7662458f4dd4cede2829537c795c40",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15997:178502"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15999:23559"
        }
      },
      scopes: [
        "STROKE_COLOR"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28129",
      name: "Stroke/Bold Tinted",
      description: "",
      key: "206a3b5aa6fe57a818c806afc559c38fa91fee4f",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8348:14418"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8640:12347"
        }
      },
      scopes: [
        "STROKE_COLOR"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28130",
      name: "Stroke/Heavy Tinted",
      description: "",
      key: "559327a148036cd6081149ee7351dfd579b4607a",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15997:178504"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15999:23561"
        }
      },
      scopes: [
        "STROKE_COLOR"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28131",
      name: "Stroke/Contrasting Tinted",
      description: "",
      key: "a2b4bda7cf6da1212e57bf9127bf84ffd807cb81",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15997:178505"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:15999:23562"
        }
      },
      scopes: [
        "STROKE_COLOR"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28132",
      name: "Appearance",
      description: "",
      key: "7d1a936038e6724fab57feded6c6e2187b3e4ed8",
      resolvedType: "STRING",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:6107:319"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8104:2722"
        }
      },
      scopes: [
        "ALL_SCOPES"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    },
    {
      id: "VariableID:16178:28133",
      name: "Default",
      description: "",
      key: "007839ff184c56e661efa52036d70798d364ea36",
      resolvedType: "COLOR",
      valuesByMode: {
        "16178:1": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8119:5944"
        },
        "16178:2": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:8119:5945"
        }
      },
      scopes: [
        "ALL_SCOPES"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:16178:28095"
    }
  ]
};
var appearance_default = {
  schemaVersion,
  collection
};
export {
  collection,
  appearance_default as default,
  schemaVersion
};
