// @marcelinodzn/ds-tokens/dist/chartbar-width-P2X4FLCD.mjs
import "./chunk-4UEJOM6W.mjs";

// src/collections/chartbar-width.json
var schemaVersion = 1;
var collection = {
  id: "VariableCollectionId:chart:barWidth",
  name: "Chart/Bar Width",
  key: "chart-bar-width-collection",
  hiddenFromPublishing: false,
  defaultModeId: "chart:barWidth:M",
  modes: [
    {
      name: "XS",
      modeId: "chart:barWidth:XS"
    },
    {
      name: "S",
      modeId: "chart:barWidth:S"
    },
    {
      name: "M",
      modeId: "chart:barWidth:M"
    },
    {
      name: "L",
      modeId: "chart:barWidth:L"
    },
    {
      name: "XL",
      modeId: "chart:barWidth:XL"
    },
    {
      name: "2XL",
      modeId: "chart:barWidth:2XL"
    }
  ],
  remote: false,
  variableIds: [
    "VariableID:chart:barWidth:0"
  ],
  variables: [
    {
      id: "VariableID:chart:barWidth:0",
      name: "barWidth",
      description: "",
      key: "a4cb50a575abd6864eb08d350f958cf430cb3042",
      resolvedType: "FLOAT",
      valuesByMode: {
        "chart:barWidth:XS": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:1:2950"
        },
        "chart:barWidth:S": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:1:2949"
        },
        "chart:barWidth:M": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:1:2948"
        },
        "chart:barWidth:L": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:1:2947"
        },
        "chart:barWidth:XL": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:1:2946"
        },
        "chart:barWidth:2XL": {
          type: "VARIABLE_ALIAS",
          id: "VariableID:1:2945"
        }
      },
      scopes: [
        "ALL_SCOPES"
      ],
      hiddenFromPublishing: false,
      codeSyntax: {},
      variableCollectionId: "VariableCollectionId:chart:barWidth"
    }
  ]
};
var chartbar_width_default = {
  schemaVersion,
  collection
};
export {
  collection,
  chartbar_width_default as default,
  schemaVersion
};
